src/components/NavApp.jsx

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import { AppBar, Toolbar, Typography } from "@mui/material";
+import SearchIcon from "@mui/icons-material/Search";
+import InputBase from "@mui/material/InputBase";
+import { styled } from "@mui/material/styles";
+import PersonIcon from '@mui/icons-material/Person';
+import Badge from "@mui/material/Badge";
+import BookmarkIcon from '@mui/icons-material/Bookmark';
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
+
+const NavApp = () => {
+
+  const Search = styled("div")(({ theme }) => ({
+    background: "white",
+    width: "530px",
+    height: "40px",
+    borderRadius: "6px",
+    opacity: "1",
+  }));
+
+  const SearchIconButton = styled("div")(({ theme }) => ({
```

Comment:
Let's move these styled-components to a separate folder. Create a folder named `styled` and put the following components inside in separated files for each component.

Reply:
DONE SIR